minio/api.py

```diff
@@ -44,7 +44,8 @@
 
 # Internal imports
 from . import __title__, __version__
-from .compat import urlsplit, range, urlencode, basestring
+from .compat import (urlsplit, queryencode,
```

Comment:
In my understanding. Change I suggested shall have no impact on 3.x. 
Because this change is actually to make everything more like 3.x 
(every string to be unicode) but within 2.x python.